server/utils/mojPagination/pageResponse.ts

```diff
@@ -0,0 +1,55 @@
+import { numberRange } from '../utils'
```

Comment:
This is a modified version of the original PageResponse, but Git/VSCode has lost the fact that it was moved.